Pages/Hotels/Project.js

```diff
@@ -0,0 +1,449 @@
+entireThankYou = '//div[@id="container"]'
+
+//TC 28
+multipleChildren = '//div[@class="uitk-layout-flex uitk-layout-flex-flex-wrap-wrap"]'
+childMinus = '[aria-label="Decrease the number of children in room 1"]'
+
+//TC-17
+backMonthButton = '[aria-label="Previous month"]'
+september = '//h2[contains(text(),"September")]'
+calenderButton = '//button[@id="date_form_field-btn"]'
+
+//TC-19
+tellUsHow = '//span[contains(text(),"Tell")]'
+hittingApply = '//button[@data-stid="apply-date-picker"]'
+hittingSearch = '//button[@id="submit_button"]'
+feedBack = '//a[contains(text(),"Share feedback")]'
+goingToLocator = "[aria-label='Going to']";
+dates = '//button[@id="date_form_field-btn"]'
+clickNextMonth = '[aria-label="Next month"]'
+
+// TC-29
+property = '//div[contains(text(),"List your property")]'
+wouldYouLike = '//p[contains(text(),"What would you like to list?")]'
+lodging = '//p[contains(text(),"Lodging")]'
+private = '//p[contains(text(),"Private residence")]'
+step1of3 = '//div[contains(text(),"Step 1 of 3")]'
+step2of3 = '//div[contains(text(),"Step 2 of 3")]'
+step3of3 = '//div[contains(text(),"Step 3 of 3")]'
+earn = '//h1[contains(text(),"See how much you could earn!")]'
+plusBedroom = '//button[@aria-label="Increase bedrooms"]'
+plusBathroom= '//button[@aria-label="Increase bathrooms"]'
+nextButton ='//button[@id="propertyInfoNextBtn"]'
+propertyLocated = '//h1[contains(text(),"Where is your property located?")]'
+enterAddress = '//input[@id="locationTypeAhead"]'
+graph = '//div[@data-wdio="google-map-component"]'
+pinInGraph = '//body/form[1]/div[1]/div[1]/div[2]/div[2]/div[2]/section[1]/div[1]/div[1]/div[1]/div[1]/section[1]/div[1]/div[1]/div[1]/div[2]/div[1]/div[4]/div[2]'
+pinUnder = '//span[contains(text(),"Move the pin to adjust the location of your proper")]'
+insideTheAuto = '//ul[@class="typeahead-prediction-list fds-field-select"]'
+
+// TC-30
+getTheApp = '//button[@id="submitBtn"]'
+phoneNumber = '//input[@id="phoneNumber"]'
+errorMessageApp = '//div[@id="phoneNumber-error"]'
+
+// TC -23
+star5 = '//span[contains(text(),"5★")]'
+sortBy = '//select[@id="sort"]'
+optionPrice ='//option[@value="PRICE_LOW_TO_HIGH"]'
+firstStarRating = '//body/div[@id="app-shopping-pwa"]/div[@id="app-layer-manager"]/div[@id="app-layer-base"]/div[1]/main[1]/div[1]/div[1]/div[1]/div[1]/div[1]/section[1]/div[1]/div[2]/div[1]/div[2]/section[2]/ol[1]/li[2]/div[1]/div[1]/div[2]/div[1]/div[1]/div[1]'
+secondStarRating = '//body/div[@id="app-shopping-pwa"]/div[@id="app-layer-manager"]/div[@id="app-layer-base"]/div[1]/main[1]/div[1]/div[1]/div[1]/div[1]/div[1]/section[1]/div[1]/div[2]/div[1]/div[2]/section[2]/ol[1]/li[3]/div[1]/div[1]/div[2]/div[1]/div[1]/div[1]'
+thirdStarRating ='//body/div[@id="app-shopping-pwa"]/div[@id="app-layer-manager"]/div[@id="app-layer-base"]/div[1]/main[1]/div[1]/div[1]/div[1]/div[1]/div[1]/section[1]/div[1]/div[2]/div[1]/div[2]/section[2]/ol[1]/li[4]/div[1]/div[1]/div[2]/div[1]/div[1]/div[1]'
```

Comment:
Change above 3 long xpath. Its not good, may break any time. Create your custom xpath.